### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]:  Name property is empty for button in "Install dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1374]
**Repro Steps:**

1. Open and Start Eclipse.
2. Navigate to Help menu.
3. Navigate and select Install new software Menu item under Help menu.
4. Install dialog will open.
5. In "Install" dialog , within the Work with text box, type http://dl.microsoft.com/eclipse/.
6. Navigate to and check Azure Toolkit for Eclipse, and uncheck Contact all update sites during install to find required software.
7. Select "Next" button.
8. Observe the name property for the button using inspect

**Actual Result:**
Name property is empty for the button

**Narrator behaviour:**
button.

**NVDA behaviour:**
toolbar
button

**Expected Result:**
Name property for the button should be "Stop".

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![stop button-inspect](https://user-images.githubusercontent.com/22956130/35680890-a57fdee8-0781-11e8-8c9c-493b8453056b.jpg)
![stop button-narrator](https://user-images.githubusercontent.com/22956130/35680893-a85651b0-0781-11e8-8775-71c178be7618.jpg)
![stop button-nvda](https://user-images.githubusercontent.com/22956130/35680896-aa580f44-0781-11e8-8e2b-b841eb498b70.jpg)

**Comment by Eskibear** [comment]
It's Eclipse IDE's widget, out of the toolkit's control.

**Comment by Eskibear** [comment]
tracked in https://bugs.eclipse.org/bugs/show_bug.cgi?id=533979

**Comment by Eskibear** [comment]
linked to #1619

**Comment by Eskibear** [comment]
* **Importance of experience**: somewhat important. It's a one-time installation.
* **Quality of experience**: not impacted. A simple workaround exists. There is another way to install the toolkit: Help -> Eclipse Marketplace... -> search for "azure toolkit" -> Install.
![market](https://user-images.githubusercontent.com/2351748/44134673-7f0f2560-a098-11e8-9b03-2036cd05b532.gif)

* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by WiproPSAccTest** [comment]
Agreed with the resolution . Can you please remove A11YExclusion, wontfix, Resolved, notreproduce labels wherever applicable and add ExclusionAgreed

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749815
